addons/web/static/src/core/domain_selector/domain_selector_operator_editor.js

```diff
@@ -53,6 +53,8 @@ export function getDomainDisplayedOperators(fieldDef) {
             return ["=", "!=", "ilike", "not ilike", "set", "not_set"];
         case "properties":
             return ["set", "not_set"];
+        case "binary":
+            return ["set", "not_set"];
```

Comment:
```suggestion
        case "binary":
        case "properties":
            return ["set", "not_set"];
```
Should use existing return